index.html

```
@@ -1 +1,54 @@
+  ```html
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)